### PR TITLE
Add TensorFlow Lite Example

### PR DESCRIPTION
**Pull request by vedantgurav**
Adds a Python example for running a TensorFlow Lite model

### CODE REVIEW

File: examples/tflite-camera/main.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    return predict
+
+
+def setup_preprocessing(cap):
+    ret, frame = cap.read()
+
+    # Creates a `normalize` function to normalize the input similarly to the training
+    means = [.485, .456, .406]
+    variances = [np.square(.229), np.square(.224), np.square(.225)]
+    normalize = Normalization(mean=means, variance=variances)
```
Comment (vedantgurav) [inline examples/tflite-camera/main.py:34]: Normalisation using a Keras class

File: examples/tflite-camera/main.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    else:
+        size = width
+        y = (height - width) / 2
+
+    # Prepares a function that will preprocess an input image for prediction
+    def preprocess(frame):
+        frame = np.array(frame)
+        center_cropped = frame[int(y): int(y + size), int(x): int(x + size)]
+        resized = cv2.resize(center_cropped, (224, 224))
+        normalized = normalize(np.divide(resized, 255))
```
Comment (vedantgurav) [inline examples/tflite-camera/main.py:55]: Using the normalized function

File: examples/tflite-camera/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+opencv-python
+numpy
+tensorflow
```
Comment (vedantgurav) [inline examples/tflite-camera/requirements.txt:3]: This didn't work for me on Mac, had to install "tensorflow-macos"

**Review by nicknikolov — APPROVED** [review]
